src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -1664,6 +1695,8 @@ HRESULT CEndlessUsbToolDlg::OnTryEndlessSelected(IHTMLElement* pElement)
 {
     FUNCTION_ENTER;
 
+	Analytics::instance()->eventTracking(_T("UI"), _T("USBType"), _T("Live"));
```

Comment:
Is it worth making the categories match the page names rather than setting them all to UI?